jekyll/_cci2/building-docker-images.md

```diff
@@ -119,8 +119,9 @@ CircleCI supports multiple versions of Docker.
 
 For **x86** architecture, the following versions are available:
 
+- `24.0.6` (default)
```

Comment:
technically they cant specify this as version it would be requested through docker-default

Reply:
how should we phrase it?

Reply:
- `default` (24.0.6)